src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final Random random = new Random();
+    public static final FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
Don't use static fields for `FigureSupplier`. Instead, instantiate `FigureSupplier` within the `main` method to avoid static context.